org.lyx.LyX.yml

```diff
@@ -0,0 +1,53 @@
+app-id: org.lyx.LyX
+runtime: org.kde.Platform
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+command: lyx-sh.sh
+rename-desktop-file: lyx.desktop
+rename-icon: lyx
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --filesystem=~/.lyx/:create
+  - --filesystem=xdg-config
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-documents
+  - --filesystem=/tmp/
```

Comment:
```suggestion
```

Reply:
The /tmp directory is used by default by the program for compilations, temporary pdfs.

Reply:
and? what happen in the sandbox stays in the sandbox.

Reply:
Without that directory, intermediate compilations could not be made, only exports could be made.

Reply:
> Without that directory, intermediate compilations could not be made, only exports could be made.

Why do you need /tmp on host? The sandbox has its own private /tmp directory.

Reply:
I didn't know that about /tpm, however I suppose that being in the isolated environment it is not possible to directly access the files found in that directory and therefore it is not possible to view the modifications that are made until when perform the export of the project.
I will try to find a solution to avoid using the /tmp directory.

Reply:
you can set `TMPDIR=/var/tmp` instead

Reply:
Using `TMPDIR=/var/tmp` is the preferred way to go.

Alternatively you can patch `lib/lyxrc.dist?`, see https://www.mail-archive.com/lyx-devel@lists.lyx.org/msg220271.html

Reply:
Also /var/tmp in sandbox is not a tmpfs, it is aliased to XDG_CACHE_HOME/tmp inside the sandbox ie. $HOME/.var/app/appid/cache/tmp

If you expect the contents of it to be gone and not be gradually filled, it won't work.